docs/src/content/docs/reference/frontmatter.md

```diff
@@ -133,3 +133,18 @@ interface HeroConfig {
   }>;
 }
 ```
+
+### `lastUpdated`
+
+**type:** `Date`
```

Comment:
I’d suggest allowing us to set an override here either with a `Date` (for an explicit override to a specific value) or a `boolean` (to enable/disable the default git-based date for specific pages). This is also how the frontmatter [`editUrl`](https://starlight.astro.build/reference/frontmatter/#editurl) option works.

```suggestion
**type:** `Date | boolean`
```

The docs below would also need a small tweak to reflect that.